docs/source/_toctree.yml

```
@@ -38,6 +38,22 @@
           title: ClickHouse
         - local: duckdb
           title: DuckDB
+          sections:
+            - local: duckdb_cli
+              title: DuckDB CLI
+              sections:
+                - local: duckdb_cli_auth
+                  title: Authentication for private and gated datasets
+                - local: duckdb_cli_select
+                  title: Query datasets
+                - local: duckdb_cli_sql
+                  title: Perform SQL operations
+                - local: duckdb_cli_combine_and_export
+                  title: Combine datasets and export
+                - local: duckdb_cli_vector_similarity_search
+                  title: Perform vector similarity search
+                - local: duckdb_cli_fts
+                  title: Implement full-text search
```

maybe it's because we don't support this depth of sections ? trying with one less level
```suggestion
        - local: duckdb
          title: DuckDB
          sections:
            - local: duckdb_cli
              title: DuckDB CLI
            - local: duckdb_cli_auth
              title: Authentication for private and gated datasets
            - local: duckdb_cli_select
              title: Query datasets
            - local: duckdb_cli_sql
              title: Perform SQL operations
            - local: duckdb_cli_combine_and_export
              title: Combine datasets and export
            - local: duckdb_cli_vector_similarity_search
              title: Perform vector similarity search
            - local: duckdb_cli_fts
              title: Implement full-text search
```

---

Added but not sure why it is still now working ..